<!--written by Daniel Ruiz, druiz@our.ecu.edu.au-->
<launch>
  <include file="$(find image_proc_fisheye)/launch/t265.launch"/>
  <!-- nodelet manager from image stream -->
  <node pkg="nodelet" type="nodelet" name="camera_nodelet_manager"  args="manager" />
  <node pkg="nodelet" type="nodelet" name="image_proc_fisheye" args="load image_proc_fisheye/RectifyNodelet camera_nodelet_manager" output="screen">
    <remap from="image_raw" to="/camera/fisheye1/image_raw" />
    <remap from="image_rect" to="/camera/color/image_rect" />
  </node>
</launch>